cves/kernel/CVE-2013-5634.yml

```diff
@@ -282,8 +294,9 @@ ipc:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    Introduced in the ARM Power State Coordination Implementation
```

Comment:
May be worth explaining what ARM Power State Coordination Implementation is / why it involves ipc